### PR TITLE
Refresh when timerange change

### LINKED ISSUE #8: The label values not correct
image

Currently, the label values (aka the select options) always get the latest values, which should be related to the selected time range. So if we delete a cluster, then we can't find its history logs.

### PR DESCRIPTION
**Pull request by baurine**
close #8 

- Re-fetch the select options when the time range changes
- Auto-generate the logcli command

https://user-images.githubusercontent.com/1284531/115225983-48b99480-a141-11eb-9867-7795e9308d49.mp4

### CODE REVIEW

File: src/ExploreQueryEditor.tsx [hunk 1]
```diff
@@ -373,6 +406,15 @@ export default function ExploreQueryEditor(props: Props) {
           />
         </InlineField>
       </div>
+      <div className="query-field">
+        <InlineField label="logcli" tooltip="logcli command, copy and run it in the command line">
```
Comment (breezewish) [inline src/ExploreQueryEditor.tsx]: I suspect people will be more confused if you display this textbox LOL

Comment (baurine) [reply src/ExploreQueryEditor.tsx]: I think so as well, will refine the wording soon.

Comment (breezewish) [reply src/ExploreQueryEditor.tsx]: Is it possible to hide this textbox? We can just provide a button named "Copy log export command".

Comment (baurine) [reply src/ExploreQueryEditor.tsx]: Sure, but maybe the user wants to modify the command before running it, for example, enlarge the limit values (default is 100000)?

**Comment by baurine** [comment]
Update, only show "Copy" button, hide the text box.

image

image

File: src/ExploreQueryEditor.tsx (hunk 1 above)
Comment (baurine) [reply src/ExploreQueryEditor.tsx]: After discussion, the text box will be removed.

**Pull request merged**